style.css

```diff
@@ -87,15 +84,54 @@ body {
   margin: 21px 40px;
 }
 
-.blue-text {
+.blue-text a {
   font-family: "Poppins", sans-serif;
   font-style: normal;
-  font-weight: 500;
-  color: #7f8cff;
+  color: blue;
   flex: none;
   order: 0;
   flex-grow: 0;
   margin: 40px 40px;
+  font-weight: 800;
+  text-decoration: none;
+}
+
+.hambu-menu {
+  display: none;
+  background-color: rgba(96, 112, 255, 1);
+  flex-direction: column;
+  position: fixed;
+  height: 100vh;
+  top: 0;
+  width: 557px;
+  align-items: center;
+  gap: 50px;
+}
+
+.hambu-list {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 50px;
+  position: relative;
+  right: 169px;
```

Comment:
- [ ] Please use best practice CSS, instead `right: 169px` you can use `text-align: center;`